packages/@aws-cdk/aws-codepipeline-actions/test/ecs/test.ecs-deploy-action.ts

```diff
@@ -80,6 +80,21 @@ export = {
 
       test.done();
     },
+
+    'can be created by ImportedBaseService'(test: Test) {
+      const service = anyIBaseService();
+      const artifact = new codepipeline.Artifact('Artifact');
+
+      test.doesNotThrow(() => {
+        new cpactions.EcsDeployAction({
+          actionName: 'ECS',
+          service,
+          imageFile: artifact.atPath('imageFile.json'),
+        });
+      });
```

Comment:
Can you instead add it to the pipeline, and then assert (using `toHaveResourceLike`) that it creates a CodePipeline Action with the expected service and cluster names?